druid_setup/single/docker-compose.yml

```diff
@@ -28,20 +35,13 @@ volumes:
   broker_var: {}
   coordinator_var: {}
   router_var: {}
-  druid_shared:
-    driver: local
-    driver_opts:
-      type: none
-      o: bind
-      device: /home/share
```

Comment:
Rather keep this and align with the other docker-compose.yml file.

Reply:
100% agree that the single and cluster compose files should match in storage layer approach. 

I actually had to switch this file from using the druid_shared network to using bind mounts to get indexing to work. If we want to switch back to using a network here, we'll have to re-test the index pipeline. Beyond this, sharing two reasons we may want to stick with bind mounts. (1) While the recommendation is for Harmony to run a dockerized pipeline, this won't necessarily be the case. If the pipeline is run directly on the host machine, then a non-Docker process will modify the storage layer (ie write to /home/share/data) and the volume won't grant access. (2) The "load data from local disk" feature in the Druid console is useful, but in order for it to work, all Druid services need access to the host filesystem. What do you think?

Reply:
Yes we do want to stick with bind mounts. Conceptually there is no difference between what there was and what you added. As is has it under `volumes` where it only has to be defined once and referenced throughout the file. What there is currently is also a bind mount (hence the `o: bind` property). You can refer to `data_output` right below that is still defined like this and has been working?